### DISCUSSION TITLE
Location data types

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2259] [Feature Ideas]
Mathesar should support location data types, including storing geographical coordinates, addresses, or subsets of addresses (cities, countries, etc.)

This could also include support for [PostGIS](https://postgis.net/) data types.

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by AJamesPhillips** [comment]
I would to be able to store [this kind of info](https://github.com/TheWorldSim/world-sim-data/blob/master/data/solarpv_capacity_summary/data/_2018_united_kingdom_loss10percent_tracking1_tilt35_azim180_month_average%40core%400.0.10.csv), that although looks like single coordinates they are actually the center of polygons describing areas that populate this data:

<img width="355" alt="Screenshot 2023-03-24 at 14 09 16" src="https://user-images.githubusercontent.com/526509/227544508-ea1d93bb-89ee-4dca-ac2a-487fa5c301a5.png">

(Screenshot from [TheWorldSim EnergyExplorer](https://theworldsim.org/sims/energy-explorer/latest/index.html))

**Reply by kgodey** [reply]
Thanks @AJamesPhillips!
